packages/evolution-backend/src/services/routing/RouteCalculationFromTransition.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+// This file contains route calculation utilities that can be used by the
+// survey. The should call chaire-lib's routing services to calculate the route.
+import { RoutingOrTransitMode } from 'chaire-lib-common/lib/config/routingModes';
+import { RouteCalculationParameter, RoutingResultByMode } from './types';
+import projectConfig from '../../config/projectConfig';
+import { URL } from 'url';
+
+// TODO Handle token expiration. For now though, the evolution app will restart more often than the token expiration
+let token: string | undefined = undefined;
```

Comment:
is there a way to name or mark this variable to say that it's global ? Can be confusing when you read the code. 
Might also need a comment to say that.

Reply:
Added a comment. And a FIXME that this file could be rewritten as a class, with the token being an attribute.